src/aap_eda/core/validators.py

```diff
@@ -132,38 +138,40 @@ def check_if_de_valid(image_url: str, eda_credential_id: int):
             % {"image_url": image_url, "tag": tag}
         )
 
-    credential = get_credential_if_exists(eda_credential_id)
-    inputs = yaml.safe_load(credential.inputs.get_secret_value())
-    credential_host = inputs.get("host")
+    if eda_credential_id != -1:
```

Comment:
Why this `-1`? Seems to me a bit odd. What I think we should do it's just make `eda_credential_id` optional

Reply:
fixed! Great point